packages/components/src/KaizenProvider/KaizenProvider.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { KaizenProvider, KaizenProviderProps } from "./KaizenProvider"
+
+const KaizenProviderWrapper = ({
+  children,
+  ...restProps
+}: KaizenProviderProps): JSX.Element => (
+  <KaizenProvider {...restProps}>{children}</KaizenProvider>
+)
+
+describe("<KaizenProvider />", () => {
+  it("renders it's children", () => {
+    const ReactApp = (): JSX.Element => <div>Hello App</div>
+    render(
```

Comment:
Wait, what's this test doing? There's no `expect` 😬

Reply:
Haha oops, for some reason I thought it would just fail if the render failed, but that doesn't even make sense cause it's running react, so of course it renders...